From fb34a9926845950eb011d47ad5c623030429396b Mon Sep 17 00:00:00 2001
From: Valentin Caron <valentin.caron@foss.st.com>
Date: Fri, 31 May 2024 17:01:44 +0200
Subject: [PATCH 0967/1695] dts: stm32: update powers compatible in
 stm32mp211.dtsi

We now dispose of new compatibles: "st,stm32mp21-pwr",
"st,stm32mp21-pwr-regu".

Use them in stm32mp211.dtsi.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git db4ed8dcb3aee32be6931bce2a7d52745ea84013 dts: stm32: update powers compatible in stm32mp211.dtsi]
Signed-off-by: Valentin Caron <valentin.caron@foss.st.com>
Change-Id: Ib66fc1859b78da355af398e287e65ca1332f0854
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/384523
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: Thomas BOURGOIN <thomas.bourgoin@foss.st.com>
Domain-Review: Patrick DELAUNAY <patrick.delaunay@foss.st.com>
---
 core/arch/arm/dts/stm32mp211.dtsi | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/core/arch/arm/dts/stm32mp211.dtsi b/core/arch/arm/dts/stm32mp211.dtsi
index ce15a0dd9..1db59e673 100644
--- a/core/arch/arm/dts/stm32mp211.dtsi
+++ b/core/arch/arm/dts/stm32mp211.dtsi
@@ -678,31 +678,31 @@
 			status = "disabled";
 
 			vdda18adc: vdda18adc {
-				compatible = "st,stm32mp25-pwr-regu";
+				compatible = "st,stm32mp21-pwr-regu";
 				regulator-name = "vdda18adc";
 				status = "disabled";
 			};
 
 			vddio1: vddio1 {
-				compatible = "st,stm32mp25-pwr-regu";
+				compatible = "st,stm32mp21-pwr-regu";
 				regulator-name = "vddio1";
 				status = "disabled";
 			};
 
 			vddio2: vddio2 {
-				compatible = "st,stm32mp25-pwr-regu";
+				compatible = "st,stm32mp21-pwr-regu";
 				regulator-name = "vddio2";
 				status = "disabled";
 			};
 
 			vddio3: vddio3 {
-				compatible = "st,stm32mp25-pwr-regu";
+				compatible = "st,stm32mp21-pwr-regu";
 				regulator-name = "vddio3";
 				status = "disabled";
 			};
 
 			vddio: vddio {
-				compatible = "st,stm32mp25-pwr-regu";
+				compatible = "st,stm32mp21-pwr-regu";
 				regulator-name = "vddio";
 				regulator-always-on;
 				status = "disabled";
-- 
2.47.2

